web/react/components/channel_info_modal.jsx

```diff
@@ -1,21 +1,13 @@
 // Copyright (c) 2015 Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
-var ChannelStore = require('../stores/channel_store.jsx');
-
-export default class CommandList extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            channel_id: ChannelStore.getCurrentId()
-        };
-    }
+const Modal = require('./modal.jsx');
 
+export default class ChannelInfoModal extends React.Component {
     componentDidMount() {
         var self = this;
         if (this.refs.modal) {
-            $(ReactDOM.findDOMNode(this.refs.modal)).on('show.bs.modal', function show(e) {
+            $(ReactDOM.findDOMNode(this.refs.modal)).on('show.bs.modal', (e) => {
```

Comment:
Oops, that function actually isn't needed at all. It's just left over from before I made my changes